packages/swap/src/utils/fees.ts

```diff
@@ -26,7 +26,10 @@ export const calculateIncludedSwapFees = async (
   const networkFeeHundredthPips = getPoolsNetworkFeeHundredthPips(
     env.CHAINFLIP_NETWORK,
   );
-  const pools = await getPools(srcAsset, destAsset);
```

Comment:
could we just return an empty array if both are usdc? bc there are no fees technically

Reply:
so changed it to return network fee only